conjure-undertow-lib/src/main/java/com/palantir/conjure/java/undertow/lib/Service.java

```diff
@@ -17,12 +17,13 @@
 package com.palantir.conjure.java.undertow.lib;
 
 /**
- * Creates a {@link Registrable} which may be registered with a web server. The server is responsible
- * for providing a {@link ServiceContext} allowing API implementors to add APIs using
+ * Registers a service interface with a web server. The server is responsible
+ * for providing a {@link ServiceContext} and {@link EndpointRegistry} allowing
+ * API implementors to add APIs using
  * <code>server.api(MyServiceEndpoints.of(myServiceImpl)</code>.
  */
 public interface Service {
```

Comment:
If we want to rename to `UndertowService`, this is the time to do it (assuming we decide to take a break here). @iamdanfox thoughts?